### PR TITLE
Various Tweaks and Medium auto-turret addition

### PR DESCRIPTION
**Pull request by ItsSamGray**
## Additions

- Added a medium auto-turret chambered in 7.62NATO to bridge the gap between the mini-turret and heavy auto-turret.

## Changes

- Rebalanced Barkskin dryad variants' armor values to be more in line with other similar animals.
- Changed manned turrets from craftable to buildable.
- Buffed SkySteel armor values to make it worthwhile as a material for armor crafting.
- Updated the bipods loading tip to fit the box better as it was too long.
- Fixed some turret bulk values.
- Changed auto-turrets to have a 5 round burst for ammo preservation.
- Added backpacks to imperial trooper and champions to fix their carry bulk problem.
- Added tribal backpacks to tribal chief pawns to fix their carry bulk problem.
- Tweaked armor glands from royalty to have a flat movement speed penalty instead of a multiplier to fix imperial stellic guards and cataphracts' carry weight problem.
- Changed Cataphract armor's weight capacity from 80 to 100kg to alleviate some carry weight issues
- Rebalanced the Minigun based on the M134D to be a bit more practical
- Rebalanced armor penetration of .44 Magnum and .454 Casull

## Reasoning

- Barkskin dryads rival thrumbos in armor values in the base game and this should be reflected to an extent. Their low HP compensates for the high armor value.
- Manned turrets in all other mods are buildable and having CE ones be craftable doesn't make sense, not consistent and isn't practical.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] Playtested a colony

### CODE REVIEW

**Review by albertbokor — DISMISSED** [review]

File: Patches/Alpha Animals/ThingDefs_Misc/AlphaAnimals_CE_Patch_Resources_Leathers.xml [hunk 1]
```diff
@@ -54,13 +54,13 @@
 				<li Class="PatchOperationReplace">
 					<xpath>/Defs/ThingDef[defName="AA_SkySteel"]/statBases/StuffPower_Armor_Sharp</xpath>
 					<value>
-						<StuffPower_Armor_Sharp>1.15</StuffPower_Armor_Sharp>
+						<StuffPower_Armor_Sharp>1.5</StuffPower_Armor_Sharp>
```
Comment (N7Huntsman) [inline Patches/Alpha Animals/ThingDefs_Misc/AlphaAnimals_CE_Patch_Resources_Leathers.xml:57]: Reduce this to 1.25

File: Patches/Alpha Animals/ThingDefs_Misc/AlphaAnimals_CE_Patch_Resources_Leathers.xml [hunk 2]
```diff
@@ -54,13 +54,13 @@
...
 					<value>
-						<StuffPower_Armor_Sharp>1.15</StuffPower_Armor_Sharp>
+						<StuffPower_Armor_Sharp>1.5</StuffPower_Armor_Sharp>
 					</value>
 				</li>
 				<li Class="PatchOperationReplace">
 					<xpath>/Defs/ThingDef[defName="AA_SkySteel"]/statBases/StuffPower_Armor_Blunt</xpath>
 					<value>
-						<StuffPower_Armor_Blunt>1.6</StuffPower_Armor_Blunt>
+						<StuffPower_Armor_Blunt>2.25</StuffPower_Armor_Blunt>
```
Comment (N7Huntsman) [inline Patches/Alpha Animals/ThingDefs_Misc/AlphaAnimals_CE_Patch_Resources_Leathers.xml:63]: Reduce this to 1.85.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
- Make a new folder in the UI section for the turret UI icons.
- Reduce your buffs to Sky Steel. In the mod, it's only about 10% stronger armor-wise than normal steel. It's chief strength over steel that it increases item HP by 50%.
- Revert your changes to heat/flammability on steel and plasteel. Those will require a larger discussion will have to be had regarding balance, and why you want to change the current vanilla stats.
- I'm onboard with the armor reduction to VFE Awakened Barkskin, but I'm a little perplexed by the buff to the standard Barkskin. They were already armored against handgun rounds, why should they now be impervious to rifle fire? Given that they're far smaller than a Thrumbo, they're harder to hit in the first place.

File: Patches/Core/ThingDefs_Misc/Weapons_Guns.xml [hunk 3]
```diff
@@ -814,6 +814,7 @@
       <ammoSet>AmmoSet_556x45mmNATO</ammoSet>
     </AmmoUser>
     <FireModes>
+      <aimedBurstShotCount>5</aimedBurstShotCount>
```
Comment (N7Huntsman) [inline Patches/Core/ThingDefs_Misc/Weapons_Guns.xml:817]: Auto turrets don't get burst fire modes--if they're capable of full auto, that's what they use. This inaccuracy and ammo consumption is an intended drawback of auto turrets vs manned ones.

File: Patches/Core/PawnKindDefs_Humanlike/PawnKinds.xml [hunk 4]
```diff
@@ -744,8 +744,8 @@
     <value>
       <li Class="CombatExtended.LoadoutPropertiesExtension">
```
Comment (N7Huntsman) [inline Patches/Core/PawnKindDefs_Humanlike/PawnKinds.xml:745]: Are these pawnkinds spawning strictly with pila, or are they sometimes spawning with bows? Cuz this isn't enough ammo for a combatant with a bow.

File: Defs/ThingDefs_Misc/Weapons_Turrets.xml [hunk 5]
```diff
@@ -158,6 +218,7 @@
         <ammoSet>AmmoSet_20x102mmNATO</ammoSet>
       </li>
       <li Class="CombatExtended.CompProperties_FireModes">
+        <aimedBurstShotCount>5</aimedBurstShotCount>
```
Comment (N7Huntsman) [inline Defs/ThingDefs_Misc/Weapons_Turrets.xml:221]: Auto turrets don't get burst fire modes--if they're capable of full auto, that's what they use. This inaccuracy and ammo consumption is an intended drawback of auto turrets vs manned ones.

File: Defs/ThingDefs_Misc/Weapons_Turrets.xml [hunk 6]
```diff
@@ -115,6 +121,60 @@
         <reloadTime>7.8</reloadTime>
         <ammoSet>AmmoSet_145x114mm</ammoSet>
       </li>
+      <li Class="CombatExtended.CompProperties_FireModes">
+        <aimedBurstShotCount>5</aimedBurstShotCount>
```
Comment (N7Huntsman) [inline Defs/ThingDefs_Misc/Weapons_Turrets.xml:125]: Auto turrets don't get burst fire modes--if they're capable of full auto, that's what they use. This inaccuracy and ammo consumption is an intended drawback of auto turrets vs manned ones.

File: Defs/ThingDefs_Misc/Weapons_Turrets.xml [hunk 7]
```diff
@@ -73,19 +73,25 @@
         <reloadTime>7.8</reloadTime>
         <ammoSet>AmmoSet_6x24mmCharged</ammoSet>
       </li>
+      <li Class="CombatExtended.CompProperties_FireModes">
+        <aimedBurstShotCount>5</aimedBurstShotCount>
```
Comment (N7Huntsman) [inline Defs/ThingDefs_Misc/Weapons_Turrets.xml:77]: Auto turrets don't get burst fire modes--if they're capable of full auto, that's what they use. This inaccuracy and ammo consumption is an intended drawback of auto turrets vs manned ones.

File: Defs/Ammo/Pistols/44Magnum.xml [hunk 8]
```diff
@@ -99,7 +99,7 @@
 		<label>.44 Magnum bullet (FMJ)</label>
 		<projectile Class="CombatExtended.ProjectilePropertiesCE">
 			<damageAmountBase>15</damageAmountBase>
-			<armorPenetrationSharp>8</armorPenetrationSharp>
+			<armorPenetrationSharp>6</armorPenetrationSharp>
```
Comment (N7Huntsman) [inline Defs/Ammo/Pistols/44Magnum.xml:102]: What info is this stat changes based upon?

File: Defs/Ammo/Pistols/454Casull.xml [hunk 9]
```diff
@@ -102,7 +102,7 @@
 		<label>.454 Casull bullet (FMJ)</label>
 		<projectile Class="CombatExtended.ProjectilePropertiesCE">
 			<damageAmountBase>20</damageAmountBase>
-			<armorPenetrationSharp>6</armorPenetrationSharp>
```
Comment (N7Huntsman) [inline Defs/Ammo/Pistols/454Casull.xml:105]: What info is this stat changes based upon?

**Review by N7Huntsman — DISMISSED** [review]
A few things for the auto turrets that need adjusted, and a few items I'm looking for some further info on.

File: Defs/Ammo/Pistols/454Casull.xml (hunk 9 above)
Comment (ItsSamGray) [reply Defs/Ammo/Pistols/454Casull.xml:105]: Realism reasons based on gel penetration data and more proper scaling

File: Defs/Ammo/Pistols/44Magnum.xml (hunk 8 above)
Comment (ItsSamGray) [reply Defs/Ammo/Pistols/44Magnum.xml:102]: Realism reasons based on gel penetration data and more proper scaling

**Review by albertbokor — APPROVED** [review]

**Pull request merged**